### ISSUE TITLE
Adding the `scripts` directory as executables during install

### ISSUE THREAD

**Issue by hayalaso** [threeML/hawc_hal#57]
## Feature Request

Adding the variable `scripts=[],` in the setup function of `setup,py` can put the scripts, such as `hal_hdf5_to_fits.py` in the path for easy use.

**Comment by grburgess** [comment]
I recently figured out how to do this and can help with the click package.

**Comment by hayalaso** [comment]
@cbrisboi I can add this, but I just want to make sure about which scripts we want to be executables. Besides `hal_hdf5_to_fits.py`, is there another one we want?

**Comment by cbrisboi** [comment]
We can add the point source script too. I dont see a reason not to, although its of somewhat limited use. Having the two scripts from the readme in there would be extremely helpful though. 

Quick thing I noticed about the `hal_hdf5_to_fits.py`. The start and end MJD's are hardcoded from when I wrote it originally. We should convert those to arguments and perhaps choose updated values for the default.
